playbooks/omevv/omevv_firmware.yml

```
@@ -0,0 +1,134 @@
+---
+- name: Immediately update the firmware of a single component for a
+    specific host
+  dellemc.openmanage.omevv.omevv_firmware:
+    hostname: "192.168.0.1"
```

```suggestion
    hostname: "xxx.xx.x.x"
```
Please update other places as required.

---

Addressed the comment